components/CreateWorkout/CreateWorkout.jsx

```diff
@@ -88,6 +90,21 @@ export default ({ navigation }) => {
     setItemState(exerciseList);
   }
 
+  const sendWorkoutToDB = (newWorkout) => {
+    // const currentUser = firebase.auth().currentUser.uid;
+    const currentUser = '68w6wWz8l5QJO3tDukh1fRXWYjD2';
+
+    const dbRef = firebase.firestore();
+    const userRef = dbRef.collection('users').doc(currentUser);
+    const workoutRef = userRef.collection('workouts');
+
+    newWorkout = JSON.parse(JSON.stringify(newWorkout, (k, v) => {
+      if (v === undefined) { return null; } return v;
+    })); // This is needed so that we can have an undefined weight and color
+
+    workoutRef.add(newWorkout);
```

Comment:
Wait jk, I think that we might need to format the data a little bit when we send it off. For example, we need the time/reps instead of duration. We also want that to be updated in the store as well